backend/docker-compose.yaml

```
@@ -47,9 +47,9 @@ services:
     ports:
       - 3001:3000
     env_file:
-      - path: .env.common.docker
+      - path: ".env.common.docker"
```

Are these quotes required or a side effect of debugging?
```suggestion
      - path: .env.common.docker
```